backend/src/db/migrations/20230326124323_alter_type_enum_in_sections.ts

```diff
@@ -0,0 +1,38 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'sections';
+
+const COLUMN_NAME = 'type';
+
+const ENUM_NAME = 'section_type';
```

Comment:
Name enum like this
Tag: @what1s1ove 
```suggestion
const ENUM_NAME = 'enum_sections_type';
```